src/test/kotlin/calculator/ExpressionTest.kt

```diff
@@ -0,0 +1,59 @@
+package calculator
+
+import org.assertj.core.api.Assertions.assertThat
+import org.assertj.core.api.Assertions.assertThatIllegalArgumentException
+import org.junit.jupiter.api.assertThrows
+import org.junit.jupiter.params.ParameterizedTest
+import org.junit.jupiter.params.provider.Arguments
+import org.junit.jupiter.params.provider.MethodSource
+import java.util.stream.Stream
+
+internal class ExpressionTest {
+
+    @ParameterizedTest
+    @MethodSource("generate")
+    fun `계산하기`(operands: MutableList<Int>, operators: MutableList<String>, actual: Int) {
+
+        val expected = Expression.calculate(operands, operators)
+
+        assertThat(expected).isEqualTo(actual)
+    }
+
+    @ParameterizedTest
+    @MethodSource("failGenerate")
+    fun `연산자가 아닌 다른 문자열`(operands: MutableList<Int>, operators: MutableList<String>, actual: Int) {
+
+        assertThatIllegalArgumentException()
+            .isThrownBy { Expression.calculate(operands, operators) }
+    }
+
+    @ParameterizedTest
+    @MethodSource("failGenerateWithEmpty")
+    fun `연산자가 더 많을 때`(operands: MutableList<Int>, operators: MutableList<String>, actual: Int) {
+        assertThrows<java.util.NoSuchElementException> { Expression.calculate(operands, operators) }
```

Comment:
```suggestion
        assertThrows<NoSuchElementException> { Expression.calculate(operands, operators) }
```
<img width="758" alt="스크린샷 2022-05-11 오전 10 23 20" src="https://user-images.githubusercontent.com/50702723/167750170-91f14ec2-b656-4fc7-bc75-bbf2d94e99ca.png">

 코틀린에도 이미 NoSuchElementException가 typeAlias로 지정되어 있습니다 😁

Reply:
어엌.. 임포트가 이상하게 되었군요~!